### ISSUE TITLE
:tabe, gt, gT not working.

### ISSUE THREAD

**Issue by geoboom** [lunixbochs/ActualVim#182]
:tabe <filename> results in:
```
ActualVim: request "read" failed: buf:3 has no view
ActualVim: request "enter" failed: buf:3 has no view
```

and sublime crashing.
Are tab-related operations supported?

**Comment by lunixbochs** [comment]
no, this is controlling a real version of neovim which has its own tabs

**Comment by geoboom** [comment]
> no, this is controlling a real version of neovim which has its own tabs

oh... so for tab related operations I should just use sublime keybindings rather than the vim ones because the vim ones aren't transmitted to sublime?

**Comment by lunixbochs** [comment]
If you poke around in the actualvim settings there are ways to modify the settings and keyboard shortcut pass through even at a vim mode granularity.
